MicaForEveryone/Services/LanguageService.cs

```diff
@@ -17,12 +17,25 @@ internal class LanguageService : ILanguageService
         public LanguageService()
         {
             SupportedLanguages = GetSupportedLanguages().ToArray();
-
-            var currentLanguage = ResourceManager.Current.DefaultContext.Languages[0];
-            CurrentLanguage = SupportedLanguages.First(
-                language => currentLanguage.StartsWith(language.LanguageTag));
+            foreach (var lstLanguage in Windows.System.UserProfile.GlobalizationPreferences.Languages)
```

Comment:
Have you tested this API in unpackaged builds?

Reply:
According to https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api It's not in the list of methods that require package identity.

Reply:
Never tried, and also I believe we can just simplify this by using LINQ.

Reply:
The conversion to `Windows.Globalization.Language` is unnecessary as `Windows.System.UserProfile.GlobalizationPreferences.Languages` already returns language tags.

Reply:
I use `Language.NativeName` to show language name in Settings UI.

Reply:
> Never tried, and also I believe we can just simplify this by using LINQ.

I agree, I'm going to do that.